docs/lib/graphqlapi/fragments/flutter/authz/10_userpool.md

```diff
@@ -0,0 +1,8 @@
+Add the following code to your app:
+
+```dart
+await amplify.addPlugin(
+    authPlugins: [AmplifyAuth()]
```

Comment:
Shouldn't this say `AmplifyAuthCognito()`. Again I'm worried that these code fragments are not tested.

Reply:
Updated example.